src/code/z_player_lib.c

```diff
@@ -3803,29 +3802,28 @@ void Player_PostLimbDrawGameplay(PlayState* play, s32 limbIndex, Gfx** dList1, G
             }
         }
     } else if (limbIndex == PLAYER_LIMB_HEAD) {
-        //! FAKE:
-        if (((*dList1 != NULL) && ((((void)0, player->currentMask)) != (((void)0, PLAYER_MASK_NONE)))) &&
+        if (((*dList1 != NULL) && ((u32)player->currentMask != PLAYER_MASK_NONE)) &&
```

Comment:
Would this match too?
```suggestion
        if (((*dList1 != NULL) && ((PlayerMask)player->currentMask != PLAYER_MASK_NONE)) &&
```

Reply:
Does not work. The real solution would be `!player->currentMask`, but to keep the comparison to the enum we need the `u32` cast (see https://github.com/zeldaret/mm/pull/1783#discussion_r1906226979)